src/projects.rs

```diff
@@ -150,6 +150,26 @@ pub fn prioritize_items(config: &Config, project_name: &str) -> Result<String, S
     }
 }
 
+/// Prioritize all unprioritized items in a project
+pub fn edit_items(config: &Config, project_name: &str) -> Result<String, String> {
```

Comment:
This would probably be a good place to test and would probably get you all the test coverage you need.

Let me know if you need help with the mocking!